public/docs/ts/latest/guide/architecture.jade

```diff
@@ -163,27 +160,23 @@ figure
 
 :marked
   A _component_ controls a patch of screen called a *view*.
-  The shell at the application root with navigation links, a list of heroes, a hero editor ...
-  they're all views controlled by components.
+  The shell at the application root with navigation links, a list of heroes, and a hero editor are all views controlled by components.
 
   You define a component's application logic&mdash;what it does to support the view&mdash;inside a class.
   The class interacts with the view through an API of properties and methods.
 
   <a id="component-code"></a>
-  A `HeroListComponent`, for example, might have a `heroes` property that returns !{_an} !{_array} of heroes
-  that it acquired from a service.
-  It might have a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list.
-  The component might be a class like this:
+  For example, this `HeroListComponent` has a `heroes` property that returns !{_an} !{_array} of heroes
+  that it acquires from a service.
+  It also has a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list.
 
 +makeExcerpt('app/hero-list.component.ts', 'class')
 :marked
   Angular creates, updates, and destroys components as the user moves through the application.
   The developer can take action at each moment in this lifecycle through optional [lifecycle hooks](lifecycle-hooks.html), like `ngOnInit()` declared above.
 .l-sub-section
   :marked
-    We may wonder who is calling the component's constructor? Who provides the service parameter?
-    For the moment, have faith that Angular will call the constructor and deliver an
-    appropriate `HeroService` when we need it.
+    Angular will call the constructor and deliver an appropriate `HeroService` when you need it.
```

Comment:
Tried a few things and then deleted this subsection. What the note refers to seems better suited to a different section in the docs, especially since the original says "you might wonder why this works - trust us, it just does". That would require a lot of explanation which would be great for another page. I could try other ways of rewording, though, if you decide to keep it.